File: app/src/main/java/de/nicidienase/chaosflix/network/StreamingService.java
package de.nicidienase.chaosflix.network;

import java.util.List;

import de.nicidienase.chaosflix.entities.streaming.LiveConference;
import io.reactivex.Observable;
import retrofit2.http.GET;

/**
 * Created by felix on 23.03.17.
 */

public interface StreamingService {

	@GET("streams/v2.json")
	Observable<List<LiveConference>> getStreamingConferences();
}

File: common/src/main/java/de/nicidienase/chaosflix/common/mediadata/network/StreamingService.kt
package de.nicidienase.chaosflix.common.mediadata.network

import de.nicidienase.chaosflix.common.mediadata.entities.streaming.LiveConference
import retrofit2.Call
import retrofit2.http.GET

interface StreamingService {

    @GET("streams/v2.json")
    fun getStreamingConferences(): Call<List<LiveConference>>
}
